frontend/cypress/integration/featureFiles/graph_manipulation.feature

```diff
@@ -0,0 +1,140 @@
+Feature: Kiali Graph page - Graph manipulation 
+
+  User opens the Graph page and manipulates the "alpha", "beta" namespace graph with buttons
+    located at the bottom of the page 
+
+  Background:
+    Given user is at administrator perspective
+    And user graphs "alpha,beta" namespaces
+
+@graph-page-manipulation
+Scenario: Check if all the buttons are usable
+  Then the toggle button "Toggle Drag" is enabled
+  And the toggle button "Zoom to Fit" is enabled
+  And the toggle button "Hide Healthy Edges" is enabled
+  And the toggle button "Hide All Edges" is enabled
+  And the toggle button "Graph Layout Default Style" is enabled
+  And the toggle button "Graph Layout Style 1" is enabled
+  And the toggle button "Graph Layout Style 2" is enabled
+  And the toggle button "Graph Layout Style 3" is enabled
+  And the toggle button "Namespace Layout Style 1" is enabled
+  And the toggle button "Namespace Layout Style 2" is enabled
+  And the toggle button "Show Legend" is enabled
+     
+@graph-page-manipulation
+Scenario: Check if the Toggle Drag Graph button can be turned off 
+  When the button "Toggle Drag" is clicked
+  Then the button "Toggle Drag" is not active
+
+@graph-page-manipulation
+Scenario: Check if the Toggle Drag Graph button can be turned on 
+  When the "Toggle Drag" is turned off 
+  And the button "Toggle Drag" is clicked
+  Then the button "Toggle Drag" is active
+
+@graph-page-manipulation
+Scenario Outline: Check if the not active by default <label> Graph button can be turned on  
+  When the button "<label>" is clicked
+  Then the button "<label>" is active
+  Examples:
+      | label |
+      | Hide Healthy Edges | 
+      | Hide All Edges | 
+      | Graph Layout Style 1 | 
+      | Graph Layout Style 2 | 
+      | Graph Layout Style 3 | 
+      | Namespace Layout Style 2 | 
+
+@graph-page-manipulation
+Scenario Outline: Check if the not active by default <label> Graph button can be turned off 
+  When the "<label>" is turned on
+  And the button "<label>" is clicked 
+  Then the button "<label>" is not active
+  Examples:
+      | label |
+      | Hide Healthy Edges | 
+      | Hide All Edges | 
+
+@graph-page-manipulation
+Scenario: Check that Hide Healthy Edges and Hide All Edges cannot be active at the same time
+  When the "Hide Healthy Edges" is turned off 
+  And the "Hide All Edges" is turned off 
+  And the button "Hide Healthy Edges" is clicked
+  And the button "Hide All Edges" is clicked
+  Then the button "Hide Healthy Edges" is not active
+  And the button "Hide All Edges" is active
+
+@graph-page-manipulation
+Scenario: Check that Hide Healthy Edges and Hide All Edges cannot be active at the same time #2
```

Comment:
nit: instead of `#2` can you add something more descriptive like to distinguish it from the first test?